src/components/ConsoleOutput.jsx

```diff
@@ -2,9 +2,18 @@ import isNil from 'lodash-es/isNil';
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import Inspector from 'react-inspector';
+import superJson from 'super-json';
```

Comment:
Same question about the footprint of `super-json`, although should be simpler to lazy-load it if it’s moved to the JSChannel handler, since that’s already an asynchronous context.